### PR TITLE
Add warning when app is empty with GPU render stage

### PR DESCRIPTION
**Pull request by rschiu**
Bug: b/148540186

### CODE REVIEW

File: gapic/src/main/com/google/gapid/views/TracerDialog.java [hunk 1]
```diff
@@ -266,6 +266,8 @@ protected void buttonPressed(int buttonId) {
           "NOTE: Mid-Execution capture for %s is experimental";
       private static final String PERFETTO_LABEL = "Profile Config: ";
       private static final String NO_GPU_PROFILING_CAPABILITY = "Warning: Selected device has no GPU profiling capability.";
+      private static final String EMPTY_APP_WITH_RENDER_STAGE =
+          "Warning: Application needs to be specified for GPU render stage data.";
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/views/TracerDialog.java]: `GPU profiling data` or `GPU slices`

File: gapic/src/main/com/google/gapid/views/TracerDialog.java [hunk 2]
```diff
@@ -732,6 +745,20 @@ private void updatePerfettoConfigLabel(Settings settings) {
         perfettoConfigLabel.requestLayout();
       }
 
+      private void updateEmptyAppWithRenderStageWarning(Settings settings) {
+        if (getSelectedDevice() == null || !getSelectedDevice().isAndroid() ||
+            getSelectedApi() == null || getSelectedApi().getType() != TraceType.Perfetto) {
+          emptyAppWarning.setVisible(false);
+          return;
+            }
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/views/TracerDialog.java]: nit: ws

Comment (rschiu) [reply gapic/src/main/com/google/gapid/views/TracerDialog.java]: Done

File: gapic/src/main/com/google/gapid/views/TracerDialog.java (hunk 1 above)
Comment (rschiu) [reply gapic/src/main/com/google/gapid/views/TracerDialog.java]: Done

**Pull request merged**